memfreeme/memfree
# Add message count and date time in search history

# Pull Request for Memfree

## Description

This pull request introduces functionality to display the creation time and the number of messages in each search history entry. The displayed time is formatted as MM/DD/YYYY, HH:MM:SS AM/PM for better readability, and the number of messages is shown alongside.

## Related Issue

- Fixes 61 #61 

## Changes Made

- [X] Added new feature (Updated `sidebar-item.tsx` to include message count and datetime)

## How to Test

1. Open SideBar to view your loaded search history
2. You will see the UI change which includes number of messages and date time for each search history

## Screenshots (if applicable)
![Screenshot 2024-10-21 235102](https://github.com/user-attachments/assets/f94c19e9-8d9f-4ca4-acab-cb1644fda9a7)

## Checklist

- [X] My code is tested and works as expected
